rustls/src/server/handy.rs

```diff
@@ -168,9 +168,9 @@ impl server::ProducesTickets for NeverProducesTickets {
 
 /// Something which always resolves to the same cert chain.
 #[derive(Debug)]
-pub(super) struct AlwaysResolvesChain(Arc<sign::CertifiedKey>);
+pub(super) struct SingleCertAndKey(Arc<sign::CertifiedKey>);
```

Comment:
Not married to this name in particular, but `AlwaysResolvesChain` seems pretty idiosyncratic to me.